UniTAS/Patcher/Patches/Preloader/StaticCtorHeaders.cs

```diff
@@ -66,6 +66,9 @@ public override void Patch(ref AssemblyDefinition assembly)
         StaticLogger.Log.LogDebug("Patching static ctors");
         foreach (var type in types)
         {
+            // remove readonly from all static fields
+            StaticLogger.Log.LogDebug($"Removing readonly from static fields");
```

Comment:
actually a better idea: print type full name to show which one its processing

Reply:
ok epic